### DISCUSSION TITLE
v1.2.7.dev6

### DISCUSSION THREAD

**Discussion by jenisys** [behave/behave#1201] [General]
* Status: Distinguish between failures and errors
* Pending steps
* Support for cucumber-expressions
* Use StepNotImplementedError (was: NotImplementedError)
* ADDED: steps.code formatter -- Shows steps with code-section.
* ...

ISSUES:

* issue #1047: Step type is inherited for generic step if possible (submitted by: zettseb)
* ...

BACKWARD INCOMPATIBLE:

* parser: No longer strips trailing colon from steps with text/table section.

<hr /><em>This discussion was created from the release <a href='https://github.com/behave/behave/releases/tag/v1.2.7.dev6'>v1.2.7.dev6</a>.</em>

**Comment by bittner** [comment]
[django-behave](https://pypi.org/project/behave-django) v1.5.0 has now been released! Django integration for Behave.

Builds on this version of Behave. Now with experimental support for [Page Objects](https://behave-django.readthedocs.io/en/stable/pageobject.html).

**Comment by rotten** [comment]
Do we have a sense as to when the `.dev6` suffix will be removed and v1.2.7 will be officially released?
